scripts/espresso_block_scanner.py

```diff
@@ -0,0 +1,111 @@
+### Simple live Espresso block scanner
+### Scan all blocks looking for transaction on each VM. 
+### For each found transaction, display the VM block containing that transaction, including inclusion proofs for transactions in the Espresso block.
+
+import urllib.request
+import json;
+import time;
+import datetime;
+import sys;
+host="localhost"
+port="50000"
+if len(sys.argv) > 1:
+	host=sys.argv[1]
+if len(sys.argv) > 2:
+	port=sys.argv[2]
+url="http://"+host+":"+port+"/availability/block/"
```

Comment:
You can write this as `f"http://{host}:{port}/availability/block/`"

Reply:
It's probably better to just accept the base URL e. g. `http://localhost:50000` as argument otherwise this won't work if we have a https URL (for example when running behind the load balancer on AWS).